doc/developer/platform/ux.md

```diff
@@ -299,17 +299,63 @@ The HTTPS server exposes two APIs:
     additional control over query execution and result fetching, a la the
     [Snowflake SQL REST API].
 
-    TODO: fully describe the output format.
+    ### Output format
 
-    TODO: design support for binding parameters.
+    The output format is a JSON object with one key, `results`, whose values is
```

Comment:
Does this need to specify that each member of `results` is associated with the index of the statement that was executed? What happens if there were many statements but one of the produced an error part way through (failing the transaction and thus not executing later statements)? Are those represented in `results` or not?

Reply:
Clarified; you cannot correlate the statements to the result indices because failed transactions only return an error, and statements after an error are not executed.